components/apimgt/org.wso2.carbon.apimgt.keymgt/src/test/java/org/wso2/carbon/apimgt/keymgt/issuers/PermissionBasedScopeIssuerTestCase.java

```diff
@@ -245,6 +270,8 @@ public void testGetAuthorizedScopesWhenUserRealmIsNotNull() throws Exception {
         Mockito.when(cacheManager.getCache("REST_API_SCOPE_CACHE")).thenReturn(cache);
```

Comment:
Shall we replace these String literals with the the Constant APIConstants.REST_API_SCOPE_CACHE since you had defined it above. Change in the other places too in this class.

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/6889/commits/560d0b70a71be6a36467e4f27267f79fa178454e